api_callable.go

```diff
@@ -0,0 +1,98 @@
+package gax
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"time"
```

Comment:
I suggest the convention of having imports from the standard library, a blank line, and then the other imports.